bot/utils/randomization.py

```diff
@@ -1,6 +1,6 @@
 import itertools
 import random
-import typing as t
+from typing import Any, Iterable
```

Comment:
`Iterable` should be imported from `collections.abc` since it's deprecated in `typing`.